### PR TITLE
remove profile images on homepage

### PR DESCRIPTION
**Pull request by ekand**
As mentioned in issue #1264, the front page currently loads
avatars from github, which can hit rate limits on the api.
This commit removes the images on from the front page

mentions #1264 

## Changes proposed

remove images from the front page

[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screen Shot 2022-05-19 at 10 06 11 PM

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1306"></a>

### CODE REVIEW

**Comment by eddiejaoude** [comment]
On hover it doesn't look quite right, see screenshot...

Screenshot 2022-05-20 at 08 15 57

I think `Avatar` is the main component being used and expects an image https://www.primefaces.org/primereact/avatar/

Maybe look at switching to `Chip` without an image https://www.primefaces.org/primereact/chip/

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Left a comment - any questions let us know

**Comment by ekand** [comment]
Thanks for the feedback. I see the issue, and I will try out the 'Chip' option.

**Comment by ekand** [comment]
Here it is with Chip instead of Avatar:
Screen Shot 2022-05-20 at 7 38 30 PM

**Comment by ekand** [comment]
Oh wait, I ended up putting a Chip inside of a Chip. I'm not sure that's quite right.

**Comment by ekand** [comment]
Well, that was chaotic. I think I'll stop at this point for now. It has chip with in chip, and hover looks right to me, though I don't have a screen shot of the mouse hover.

Screen Shot 2022-05-20 at 7 55 10 PM

Would you like me to squash my commits and force push, or leave the commits as is?

File: src/Components/Home/Users.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ function Users({ list }) {
                 className="m-2 w-16rem px-3 py-2 transition-all transition-duration-300"
                 template={
                   <>
-                    <Avatar
-                      image={user.avatar}
+                    <Chip
```
Comment (emmalearnscode) [inline src/Components/Home/Users.js:83]: Great work so far. I think we can go a little further in order to simplify this as much as possible and get rid of any unused code so would you be able to update the template to just this:

`template={
                  <span className="text-overflow-ellipsis white-space-nowrap overflow-hidden">{user.name}</span>
                }`

Also, it's fine to delete the unused imports Avatar, and also Badge and Utils once the template is updated as that keeps the file as clean as possible and reduces tech debt. Let us know if you have any questions.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: src/Components/Home/Users.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { Chip } from 'primereact/chip'
-import { Avatar } from 'primereact/avatar'
+// import { Avatar } from 'primereact/avatar'
```
Comment (emmalearnscode) [inline src/Components/Home/Users.js:5]: Please delete unused imports 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

**Comment by ekand** [comment]
@emmalearnscode Thanks for the feedback. I'll make those changes now.

**Comment by ekand** [comment]
Here it is with the simplified template, as @emmalearnscode suggested.

Screen shot:
Screen Shot 2022-05-21 at 2 17 09 PM

**Review by emmalearnscode — APPROVED** [review]
Looks like it works well! Thank you for your help!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by eddiejaoude — APPROVED** [review]
Looks great! Awesome collaboration 🔥

**Pull request merged**